client/src/App.tsx

```diff
@@ -29,7 +30,9 @@ function App(): JSX.Element {
                 <Route exact path="/signup" component={Signup} />
                 <Route exact path="/dashboard" component={Dashboard} />
                 <Route path="/profile/settings" component={Settings} />
+                <Route exact path="/profile/settings/Availability" component={Availability} />
```

Comment:
So the `Settings` component renders sub pages under neath it so I would take a look at `Settings.tsx` and move the availability component there.